internal/auth0/tenant/resource_test.go

```diff
@@ -100,7 +97,6 @@ resource "auth0_tenant" "my_tenant" {
 	enabled_locales         = ["en", "de", "fr"]
 
 	flags {
-		universal_login                        = true
```

Comment:
Hm... I think the universal_login flag should probably stay, it's just the block that needs to get removed.

Reply:
🤔